CEGameApp.API/Startup.cs

```diff
@@ -24,6 +27,19 @@ public void ConfigureServices(IServiceCollection services)
                 (Configuration.GetConnectionString("DefaultConnection")));//this will be changed when uploading to azure
             services.AddControllers();
             services.AddCors();
+            services.AddScoped<IAuthRepository, AuthRepository>();
```

Comment:
I will leave more feedback in the main review but this is the same place where you should create some more domain layer classes that will take care of most of the logic in your app.